ZukoAzula2016/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//src/org/team1540/zukoazula/RobotTemplate.java=UTF-8
```

Comment:
Tyler can add whatever he wants to the .gitignore. If he wants to keep this file locally by adding it to the .gitignore, that's fine by me. I posted that comment because it's not important and the conversation needed to stop, because it doesn't matter.